README.md

```diff
@@ -1,3 +1,8 @@
 # Pandora and Titan 2018
 
 Team 1540's robot-specific code for our 2018 robots, Pandora and Titan. Does not include libraries reused from year-to-year; those can be found in [flamingchickens1540/ROOSTER](https://github.com/flamingchickens1540/ROOSTER).
+
+#### Copilot
+![Copilot Controls](img/copilot_controller.png)
```

Comment:
Should we make any effort to keep that image up-to-date or just remove it until we have final controls?